extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
+ 	bottom: 0;
+ 	left: 0 !important;
+ 	z-index: 1;
+ 	width: 100%;
+ 	display: none;
+ 	max-height: 100px;
```

Comment:
@zhouyx @sriramkrish85 `100px` is not the smaller value than 1/6 of viewport in landscape mode. Does it mean we allow more than 1/6 in landscape? I'm ok if that's the case, but I need to know what it is.